docs/pages/docs/build-platforms/github-actions.mdx

```diff
@@ -78,12 +90,14 @@ If you are having problems make sure you have done the following:
 ### Running With Branch Protection
 
 GitHub actions require a little more setup to use `auto` with branch protection.
+We can't use the `GITHUB_TOKEN` to commit to the branch because it does not have high enough permissions.
+You will need to create a token with admin access to your repo and modify your build config like the following:
 
 ```yml
 steps:
   - uses: actions/checkout@v2
     with:
-      # Ensure that git uses your token with write access to the repo
+      # Ensure that git uses your token with admin access to the repo
       token: ${{ secrets.GH_TOKEN }}
```

Comment:
I'd suggest changing this to something other than the old default `GH_TOKEN`, just for clarity. Perhaps:
```
token: ${{ secrets.ADMIN_TOKEN }}
```

Reply:
Good callout!